Document patterns for secure api keys (#18)
lacework/chef-lacework: README.md

- `node['chef-lacework']['deb_package']['package_name'] = 'lacework_latest_amd64.deb'` - Name of `.deb` package to install on Debian/Ubuntu Linux
- `node['chef-lacework']['deb_package']['shasum'] = ''` = sha256 sum of `.deb` package
- `node['chef-lacework']['rpm_package']['url'] = ''` - URL to download `.rpm` package to install the Lacework agent
- `node['chef-lacework']['rpm_package']['package_name'] = 'lacework-latest-1.x86_64.rpm'` - Name of `.rpm` package to install on RHEL/Centos/SUSE Linux
- `node['chef-lacework']['rpm_package']['shasum'] = '' ` = sha256 sum of `.rpm` package

Patterns for protecting Tokens
=======

> Due to the way the cookbook is written we are exposing all the configuration as a namespace in chef. However there are sensitive properties such as the access key.

Reviewer: The wording on this sounds opinionated. It could be reworded to sound more matter of fact. A possible suggestion.
```suggestion
All of the configuration is exposed as a namespace in chef. There are sensitive properties such as the access key.
```
